docs/hosting/hosting-cli-commands.md

```diff
@@ -56,25 +56,8 @@ To redeploy or update your app, navigate to the project directory and type `refl
 
 All the `reflex` commands come with a help manual. The help manual lists additional command options that may be useful. You type `--help` to see the help manual. Some commands are organized under a `subcommands` series. Here is an example below. Note that the help manual may look different depending on the version of `reflex` or the `reflex-hosting-cli`.
 
-```python eval
-doccmdoutput(
-    command="reflex deployments --help",
-    output="""Usage: reflex deployments [OPTIONS] COMMAND [ARGS]...
-
-  Subcommands for managing the Deployments.
-
-Options:
-  --help  Show this message and exit.
-
-Commands:
-  build-logs  Get the build logs for a deployment.
-  delete      Delete a hosted instance.
-  list        List all the hosted deployments of the authenticated user.
-  logs        Get the logs for a deployment.
-  regions     List all the regions of the hosting service.
-  status      Check the status of a deployment.
-"""
-)
+``` bash
+reflex deployments --help
```

Comment:
I thought `deployments` was replaced by `apps` ?

Reply:
it is for the new hosting service but i dont think these commands are relating to v1. these are still v0

tom has a pr up for the new v1 commands

Reply:
I just fixed those pages because the old codeblock was spilling over main content

Reply:
toms pr will overide it

Reply:
understood